thamos/lib.py

```diff
@@ -68,7 +69,10 @@ def wrapper(*args, **kwargs):
         config.verify_ssl = thoth_config.tls_verify
 
         start = monotonic()
-        result = func(ApiClient(configuration=config), *args, **kwargs)
+        api_client = ApiClient(configuration=config)
+        # Override default user-agent.
+        api_client.user_agent = f"Thamos/{thamos_version}"
```

Comment:
can we also include the operating system version?

Reply:
Adjusted, thanks!

Example of an user-agent (my machine):

```
Thamos/0.4.2 (Python 3.6.8; Linux 4.19.15-300.fc29.x86_64)
```